sdk/helper/consts/agent.go

```diff
@@ -3,3 +3,7 @@ package consts
 // AgentPathCacheClear is the path that the agent will use as its cache-clear
 // endpoint.
 const AgentPathCacheClear = "/agent/v1/cache-clear"
+
+// VaultRequestHeader is the name of the header entry used by the Agent for
+// SSRF protection.
+const VaultRequestHeader = "Vault-Request"
```

Comment:
Another place to consider: sdk/helper/consts/consts.go.  We already have other Vault-specific HTTP headers there, so it seems more natural.

They all seem to use the `X-Vault-` prefix, maybe for consistency we should do the same?

Reply:
`X-Vault-Request` is a good idea, and something I hadn't considered.  Hmm.

Reply:
+1 to `X-Vault-Request`

Reply:
I changed it to `X-Vault-Request`.